src/main/java/de/sakpaas/backend/service/SearchService.java

```diff
@@ -53,6 +55,17 @@ public SearchResultObject search(String query, CoordinateDetails coordinateDetai
   }
 
 
+  /**
+   * Creates a new SearchRequest Object.
+   *
+   * @param query             The SearchQuery
+   * @param coordinateDetails The SearchCoordinates
+   * @return A new SearchRequest
+   */
+  protected SearchRequest createRequest(String query, CoordinateDetails coordinateDetails) {
```

Comment:
Wofür brauchen wir diese Methode?

Reply:
Die Methode wandelt den Query und die Koordinaten in das SearchRequest Objekt um. Ich habe die erstellt damit dies nicht in der Search Methode gemacht werden muss. Dadurch wird es einfacher zum Testen und übersichtlicher